app/Http/Controllers/Api/VaultController.php

```diff
@@ -42,6 +53,9 @@ public function show(Request $request, Vault $vault): JsonResource
     /**
      * Create a vault.
      */
+    #[ResponseFromApiResource(JsonResource::class, Vault::class, status: 201)]
+    #[BodyParam('name', description: 'The name of the vault. Max 255 characters.')]
+    #[BodyParam('description', description: 'The description of the vault. Max 65535 characters.', required: false)]
```

Comment:
This is not true actually. The migration sets the description field to 255 characters by default.